### PR TITLE
Fixes #15040: Upgrade to ZIO rc5

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/15040

### CODE REVIEW

File: webapp/sources/utils/src/test/scala/com/normation/ZioCommonsTest.scala [hunk 1]
```diff
@@ -26,6 +26,9 @@
 package com.normation
 
 
+import java.util.regex.Pattern
```
Comment (VinceMacBuche) [inline webapp/sources/utils/src/test/scala/com/normation/ZioCommonsTest.scala]: Why do you need this imports ?

File: webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/queries/TestQueryProcessor.scala [hunk 2]
```diff
@@ -857,6 +857,8 @@ class TestQueryProcessor extends Loggable {
       ] }
       """ :: Nil
 
+
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/queries/TestQueryProcessor.scala]: Do you wanted to add those lines ?

File: webapp/sources/utils/src/test/scala/com/normation/ZioCommonsTest.scala (hunk 1 above)
Comment (fanf) [reply webapp/sources/utils/src/test/scala/com/normation/ZioCommonsTest.scala]: sequel of unfructful tests :)

File: webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/queries/TestQueryProcessor.scala (hunk 2 above)
Comment (fanf) [reply webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/queries/TestQueryProcessor.scala]: Does it matter?

Comment (VinceMacBuche) [reply webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/queries/TestQueryProcessor.scala]: no :) just jealous of your github stat increase ;)

**Comment by fanf** [comment]
Commit modified

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**